docs/source/usage.rst

```diff
@@ -0,0 +1,185 @@
+
+Sphinx object inventories can be used to create deep links between API
+documentation generated by pydoctor and by Sphinx.
+
+Linking to external API docs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It can link to external API documentation using a Sphinx objects inventory
+with the following cumulative configuration option::
+
+    --intersphinx=http://sphinx-doc.org/objects.inv
+
+Linking to your API docs
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+pydoctor's HTML generator will also generate a Sphinx objects inventory
+using the following mapping:
+
+* packages, modules -> py:mod:
+* classes -> py:class:
+* functions -> py:func:
+* methods -> py:meth:
+* attributes -> py:attr:
+
+To use this mapping in Sphinx, configure the `intersphinx extension`__::
+
+    intersphinx_mapping = {
+        'pydoctor': ('http://domain.tld/api', None),
+    }
+
+__ https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html
+
+And use external references::
+
+    :py:func:`External API <pydoctor:pydoctor.model.Documentable.reparent>`
+
+    :py:mod:`pydoctor:pydoctor`
+    :py:mod:`pydoctor:pydoctor.model`
+    :py:func:`pydoctor:pydoctor.driver.getparser`
+    :py:class:`pydoctor:pydoctor.model.Documentable`
+    :py:meth:`pydoctor:pydoctor.model.Documentable.reparent`
+    :py:attr:`pydoctor:pydoctor.model.Documentable.kind`
+
+Customize builds
+----------------
+
+.. warning:: PyDoctor does not have a stable API yet. Custom builds are prone to break. Nevertheless, `PyDoctor's API documentation is available here <api/>`_. 
+
+Use a custom System class
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can subclass the :py:class:`pydoctor:pydoctor.zopeinterface.ZopeInterfaceSystem` and pass your custom class dotted name with the ``--system-class`` argument. 
+
+System class allows you to dynamically show/hide classes or methods. 
+This is also used by the Twisted project to handle deprecation.
+
+See the `Twisted custom class documentation <https://twistedmatrix.com/documents/current/api/twisted.python._pydoctor.TwistedSystem.html>`_. Naviguate to the source code for a better overview.
+
+.. note:: Not fully documented, prone to break
+
+Use custom HTML templates
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Currently, custom HTLM templates needs to be handled with some "monkeypatch" that will selectively use the appropriate templates.
+
+See the `Twisted custom class documentation <https://twistedmatrix.com/documents/current/api/twisted.python._release.APIBuilder.html>`_. Naviguate to the source code for a better overview.
+The key thing is to apply a patch to the :py:func:`pydoctor:pydoctor.templatewriter.util.templatefile` function. 
+
+.. note:: Not fully documented, prone to break
+
+Use a custom HTML writer
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can subclass the :py:class:`pydoctor:pydoctor.templatewriter.writer.TemplateWriter` and pass your custom class dotted name with the ``--html-writer`` argument. 
+
+.. note:: Not fully documented, prone to break
+
+Full help
+---------
+
+.. include:: help.txt
```

Comment:
Not sure if we should show the pydoctor help here.... I guess that anyone can run pydoctor --help and see the options.

But I can undestand that it can help come people discover pydoctor features without having to install it.

So is OK to have it... but maybe have it as a top level RST page.

This page is already long

Reply:
I'll put it on a top level page `help.rst`